source/includes/v1/query_language_and_geo_filtering.md

```diff
@@ -1,11 +1,11 @@
 # Query Language and Geo Filtering
 
-Filtering features are built in the core of Opendatasoft API engine. Many of the previously listed APIs can take as a
-parameter filters for constraining the list of returned datasets or records.
+Filtering features are built in the core of the Opendatasoft API engine. Many of the previously listed APIs can take as
+parameters filters for constraining the list of returned datasets or records.
```

Comment:
Based on @NTerpo's comment, the following rewording makes more sense.
```suggestion
Filtering features are built in the core of the Opendatasoft API engine. Many of the previously listed APIs can take filters as parameters, so that the response only contains the datasets or records you want.
```